test/per_feature_flags_test.go

```diff
@@ -48,7 +48,7 @@ const (
 
 var (
 	alphaFeatureFlags = []string{"enable-param-enum", "enable-step-actions", "keep-pod-enabled-cancel", "enable-cel-in-whenexpression", "enable-artifacts"}
```

Comment:
We shall remove it from alphaFeatureFlags then.

Reply:
Agreed! @ericzzzzzzz

Reply:
thanks! updated!